### DISCUSSION TITLE
RTX2080 faster than Coral TPU?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#10146] [General]
I am running two frigate machines, one with an Nvidia RTX 2080 and one with a Coral TPU. The performance is great on both, however I was surprised to see that my "old" 2080 is outperforming the dedicated TPU, is that expected? The coral is inferencing at 8ms, while the RTX is going at 4ms:

### Coral
![image](https://github.com/blakeblackshear/frigate/assets/118814/a4e70c3c-b865-4702-84d4-6ecb41cd78d5)

### RTX 2080
![image](https://github.com/blakeblackshear/frigate/assets/118814/5a2d32f1-3c9c-4387-91c8-478b9e3d792b)

The systems are completely different in terms of cpu, memory and everything else. Is that playing a part? I also seem to have an "error-gpu" message on the 2080 - could that possibly have an effect on the acocunting? (Is it normal to have this error as the message suggests?)

**Comment by NickM-27** [comment]
Think about the difference in power draw between the coral and GPU, it's both unexpected that the GPU is a bit faster but it's also not nearly as efficient.

**Reply by NickM-27** [reply]
The error GPU is probably because you have a typo for hwaccel args somewhere.

**Reply by thinkloop** [reply]
right again 👍
